### ISSUE TITLE
"space" not a reserved word in Postgres

### ISSUE THREAD

**Issue by osvenskan** [sqlfluff/sqlfluff#926]
The following SQL generates `L029 | Keywords should not be used as identifiers` due to the use of the alias `space`. 

```sql
SELECT 
    foo AS space
FROM 
    bar
;
```

`space` has no special status under Postgres: https://www.postgresql.org/docs/current/sql-keywords-appendix.html

I can submit a PR to silence L029 for `space` in the Postgres dialect, but first I wanted to make sure that doing so would be viewed as an enhancement. 🙂

**Comment by barrywhart** [comment]
We had a similar discussion in #315. I think we eventually decided not to change L029 because users can configure their desired behavior in `.sqlfluff`. If that's not documented, it might be good to do so -- that could help avoid future confusion others.

**Comment by osvenskan** [comment]
It'd be great if we can silence the comments about `space` simply via altering `.sqlfluff`. I can't see how to use the config to disable L029 for a single value, though. Any pointers?

**Comment by barrywhart** [comment]
Ah yes, in the discussion on the earlier issue (which I had opened), I decided I was happy to simply disable L029 completely. Do you see value in enabling the rule but excluding certain values? If so, that would be an enhancement.

**Comment by osvenskan** [comment]
I see value for us. We use `space` frequently, and without this enhancement we'd have to either add lots of `noqa`s or disable L029 completely, neither of which is appealing. Thanks for the context. I'll work on a PR. I'm pretty sure it's a one line change.
